graph/history.go

```diff
@@ -1,24 +1,21 @@
 package graph
 
 import (
-	"encoding/json"
 	"fmt"
 	"strings"
 
 	"github.com/docker/docker/api/types"
-	"github.com/docker/docker/engine"
 	"github.com/docker/docker/image"
 	"github.com/docker/docker/utils"
 )
 
-func (s *TagStore) CmdHistory(job *engine.Job) error {
-	if n := len(job.Args); n != 1 {
-		return fmt.Errorf("Usage: %s IMAGE", job.Name)
+func (s *TagStore) CmdHistory(name string) ([]*types.ImageHistory, error) {
+	if name == "" {
+		return nil, fmt.Errorf("Usage: docker history [OPTIONS] IMAGE")
```

Comment:
This is sorta weird error. It is about old jobs syntax. I think it is okay to return error right from `s.LookupImage`.